content/en/docs/collector/transforming-telemetry.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Transforming telemetry
+weight: 31
+---
+The OpenTelemetry Collector is a convenient place to transform data before sending it to a vendor or other systems. This is frequently done for data quality, goveranance, cost, and security reasons.
+
+Processors available from the the [Collector Contrib repository](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor) support dozens of different transformations on metric, span and log data. The following sections provide some basic examples on getting started with a few frequently-used processors.
+
+The configuration of processors, particularly advanced transformations, may have a signficant impact on collector performance.
+
+## Basic filtering
+
+**Processor**: [filter processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor)
+
+The filter processor allows users to filter telemetry based on `include` or `exclude` rules. Include rules are used for defining "allow lists" where anything that does _not_ match include rules is dropped from the collector. Exclude rules are used for defining 
+"deny lists" where telemetry that matches rules is dropped from the collector.
+
+For example, to _only_ allow span data from services app1, app2, and app3 and drop data from all other services:
+
+```
```

Comment:
Add a code-block language as appropriate, here and elsewhere in the document:

```suggestion
```yaml
```

Reply:
> Add a code-block language as appropriate, here and **_elsewhere_**

You'll also need to add languages to the other code blocks. Marking this as unresolved until then. Thanks.

Reply:
@smithclay please resolve the conversation if this has been addressed